search/indexing_api_test.py

```diff
@@ -249,12 +240,12 @@ def test_index_post_with_comments(
     post = client.get_post.return_value
 
     serialize_post_mock.assert_any_call(post)
-    sync_post_mock.assert_any_call(serialize_post_mock.return_value)
-
     serialize_comments_mock.assert_any_call(post)
-    sync_comments_mock.assert_any_call(serialize_comments_mock.return_value)
 
     post.comments.replace_more.assert_called_once_with(limit=None)
+
+    print(bulk_mock.call_args_list)
+    assert bulk_mock.call_count == 4
```

Comment:
Can you include a comment about why we're expecting 4 calls? I'm assuming it's because of the 2 object types * 2 aliases each

Reply:
This was also missing an assertion on the two other calls so I added that.